### PR TITLE
Client for http added

### PR DESCRIPTION
**Pull request by mish15**
Simplifies connecting to a docconv server or container over http

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -12,83 +15,98 @@ import (
...
 		return nil, err
 	}
-	err = writer.Close()
-	if err != nil {
+	if err := w.Close(); err != nil {
 		return nil, err
 	}
 
-	req, err := http.NewRequest("POST", c.endpoint, body)
+	req, err := http.NewRequest("POST", fmt.Sprintf("http://%v/convert", c.endpoint), buf)
```
Comment (mish15) [inline client/client.go]: This locks to http, not a good idea

File: client/client.go [hunk 2]
```diff
@@ -12,83 +15,98 @@ import (
 	"github.com/sajari/docconv"
 )
 
-var (
-	defaultParam    = "input"
```
Comment (mish15) [inline client/client.go]: the param is configurable in the service, it needs to be kept

Comment (dhowden) [inline client/client.go]: Ah, I'm not looking in the right place then.  I thought this was it:

https://github.com/sajari/docconv/blob/master/docd/main.go#L112

File: client/client.go (hunk 1 above)
Comment (dhowden) [inline client/client.go]: The service in https://github.com/sajari/docconv/blob/master/docd/main.go is currently HTTP only at the moment, can easily break out the pattern to be overridden though.

File: client/client.go (hunk 2 above)
Comment (mish15) [inline client/client.go]: hmmm might be confusing with an old version. We def set this on v9. Anyway all good

File: client/client.go (hunk 1 above)
Comment (mish15) [inline client/client.go]: yep, but that assumes it won't be behind an https LB, which it is for us now

**Pull request merged**